### PR TITLE
KEYCLOAK-7471 Improve adapter documentation for parameters-forwarding…

### PR DESCRIPTION
**Pull request by mposolda**
… and prompt

### CODE REVIEW

File: securing_apps/topics/oidc/java/params_forwarding.adoc [hunk 1]
```diff
@@ -19,7 +19,11 @@ The supported parameters are:
 
 * scope
 
-* prompt
+* prompt - Values supported by {project_name} are currently:
+** `login` - SSO will be ignored and {project_name} login page will be always shown even if user is already authenticated
+** `none` - Login page will never be shown, but user will be redirected back to the application with the error in case that he
+is not yet authenticated. This allows you for example to create filter/interceptor on the application side and show some custom error page to the user.
+See more details in the specification.
```
Comment (matthewhelmke) [inline securing_apps/topics/oidc/java/params_forwarding.adoc]: * prompt - {project_name} supports these settings:
** `login` - SSO will be ignored and the {project_name} login page will be always shown, even if the user is already authenticated
** `none` - The login page will never be shown; instead the user will be redirected to the application, with an error if the user is not yet authenticated. This setting allows you to create a filter/interceptor on the application side and show a custom error page to the user.
See more details in the specification.

File: securing_apps/topics/oidc/java/params_forwarding.adoc [hunk 2]
```diff
@@ -30,3 +34,7 @@ The supported parameters are:
 Most of the parameters are described in the http://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint[OIDC specification].
 The only exception is parameter `kc_idp_hint`, which is specific to {project_name} and contains the name of the identity provider to automatically use.
 For more information see the `Identity Brokering` section in link:{adminguide_link}[{adminguide_name}].
+
+WARNING: Currently if you open the URL with the attached parameters, the adapter won't redirect you to the {project_name} in case that you
+are already authenticated in the application. For example opening `$$http://myappserver/mysecuredapp?prompt=login$$` won't automatically redirect you to the {project_name}
+login page in case that you are already authenticated to the application `mysecredapp` . This behaviour can be changed in the future.
```
Comment (matthewhelmke) [inline securing_apps/topics/oidc/java/params_forwarding.adoc]: WARNING: If you open the URL using the attached parameters, the adapter will not redirect you to {project_name} if you are already authenticated in the application. For example, opening `$$http://myappserver/mysecuredapp?prompt=login$$` will not automatically redirect you to the {project_name} login page if you are already authenticated to the application `mysecredapp` . This behavior may be changed in the future.

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
I have left some suggested edits.

**Comment by mposolda** [comment]
@matthewhelmke I've updated PR per your suggestions. Thanks!

**Review by matthewhelmke — APPROVED** [review]

**Comment by matthewhelmke** [comment]
Thank you. LGTM

**Comment by mposolda** [comment]
@matthewhelmke Thanks!

**Pull request merged**